src/braket/experimental/autoqasm/instructions/qubits.py

```diff
@@ -58,11 +61,8 @@ def _get_physical_qubit_indices(qids: list[str]) -> list[int]:
     return braket_qubits
 
 
-def _global_qubit_register(qubit_idx_expr: Union[int, str]) -> str:
-    # TODO: We should index into a oqpy.Qubit register rather
-    # than manually generating the string to index into
-    # a hard-coded global qubit register.
-    return f"{constants.QUBIT_REGISTER}[{qubit_idx_expr}]"
+def _global_qubit_register(qubit_idx_expr: Union[int, str]) -> oqpy.Qubit:
+    return oqpy.Qubit(f"{constants.QUBIT_REGISTER}[{qubit_idx_expr}]", needs_declaration=False)
```

Comment:
I removed the "TODO" comment because the oqpy qubit register is not actually created/declared at this point (that happens near the end of the program creation process), and I don't think there's a better way to index into it than what we're doing here.